ansible/roles/test/files/tools/loganalyzer/loganalyzer.py

```diff
@@ -354,15 +360,19 @@ def analyze_file(self, log_file_path, match_messages_regex, ignore_messages_rege
                     in_analysis_range = False
                     break
 
-            if in_analysis_range :
+            if in_analysis_range or is_sairedis_rec:
+                # Skip long logs in sairedis recording since most likely they are bulk set operations for non-default routes
+                # without much insight while they are time consuming to analyze
+                if is_sairedis_rec and len(rev_line) > 1000:
```

Comment:
Will this ever cause use to skip log lines with insight? Also, instead of checking the length, can we check for regex in `rev_line`?

Reply:
This would happen in the scenario of huge bulk operations. At this stage, we only care about the first default route set and that should be a set operation while other routes should be in create operations at the time. Most likely this does not skip useful information. The reason not to use regex is that the lines could be very long (with thousands of routes), and regex takes hours to parse the lines. Therefore, I use the length as a workaround at this stage.